assets/api-swagger/swagger.json

```diff
@@ -194,6 +194,66 @@
               "type": "string",
               "default": "0"
             }
+          },
```

Comment:
Goes for most of the pull requests, but these shouldn't be here, they're in their separate PR.